### DISCUSSION TITLE
Obscure Player IP

### DISCUSSION THREAD

**Discussion by digitalcrowd** [Zrips/CMI#1858] [Feature Requests]
As OP/Owner, I have some accounts I use for testing and playing with so I can take a breather from being the Owner and just be a normal player and experience what other players experience on the server.

However, I would like to obscure these accounts from being associated with my IP address. As such, I can't give Staff access to players IP information or allow them to see other players sharing the same IP address, which could be very helpful.

For example, in config you could have...

### OVERRIDE PLAYER IP INFORMATION
- playername:fakeipaddress
- Notch:210.83.11.223

OR

- playerUUID:IP (this would allow any name changes to be automatically de-associated)

This would be retroactive when enabled and obscure any IP addresses recorded for the player in database previously and make sure they are not associated with other players who are not obscured sharing the same IP.

There maybe be some associations in addition to this I am not thinking of that would also need to be obscured, but I think that should do it.

**Comment by mrfloris** [comment]
Negate the info with ip permissions so they can't see it. it's what i've done.

**Comment by digitalcrowd** [comment]
Negate the particular player you want to hide?

If possible, that is a workaround. But if some staff are savvier than others, they may wonder why a given player has info not displaying for a specific player but not anyone else.
